### DISCUSSION TITLE
classroom autograde Java

### DISCUSSION THREAD

**Discussion by shaharyi** [community/community#20932] [GitHub Education] [answered]
Did anyone succeed in getting varying grades besides 0 and 100?
I tried to set two tests, one for 80 points and another for 20 points. 
But if any of them fails, the score is 0.
If they both pass, the score is 100.
I used "Java test" with gradle like in the official example repository.

**Comment by jfullstackdev** [comment]
you mean auto-grade is to compute automatically ? like there is the code to compute the average from the given grades ?

**Reply by shaharyi** [reply]
No, I mean that Github Classroom has auto-grading feature, but it doesn't work well for me.

**Comment by shaharyi** [comment, marked as answer]
I went through Gradle CLI docs.
You have to specify the test class/es in the "run command":
gradle test --tests MyTestClass
